kong/plugins/aws-lambda/v4.lua

```diff
@@ -109,7 +109,7 @@ local function prepare_awsv4_request(tbl)
       return nil, "either 'signing_key' or 'secret_key' must be provided"
     end
   end
-  local timestamp = tbl.timestamp or os.time()
+  local timestamp = tbl.timestamp or ngx.time()
```

Comment:
Only thing I would do is to move this variable declaration closer to its usage (aka a few lines down).